pkg/converter/convert.go

```diff
@@ -54,75 +78,93 @@ func Convert(o Options) error {
 	isAzureCLI := o.TokenOptions.LoginMethod == token.AzureCLILogin
 	isWorkloadIdentity := o.TokenOptions.LoginMethod == token.WorkloadIdentityLogin
 	isAlternativeLogin := isMSI || isAzureCLI || isWorkloadIdentity
-
 	for _, authInfo := range config.AuthInfos {
+
 		if authInfo != nil {
-			if authInfo.AuthProvider == nil || authInfo.AuthProvider.Name != azureAuthProvider {
-				continue
-			}
+
```

Comment:
i think this is where we should check: is it legacy aad auth or is it exec using kubelogin? if neither, we return and do nothing